### PR TITLE
Chore/setup seo

### PR DESCRIPTION
**Pull request by AmmDuncan**
- Add meta tags from poll pages
- Add sitemap
- Add favicon
- Add open graph image

### CODE REVIEW

File: app/helpers.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { db } from "@/db/client";
+
+export async function getPolls() {
```
Comment (rob-gordon) [inline app/helpers.ts:3]: You may want to call this getPublicPolls so it's clear what the goal of it is

File: app/sitemap.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    const pollPages = polls.map((poll) => ({
+      url: `${getBaseUrl()}/poll/${poll.slug}`,
+      priority: 0.5,
+      lastModified: new Date(poll.created_at!),
+    }));
+    const staticPages = [
+      {
+        url: `${getBaseUrl()}/`,
+        priority: 1,
+        lastModified: new Date("2024-05-15"),
```
Comment (rob-gordon) [inline app/sitemap.ts:17]: Maybe replace this with the most recent public poll date since new public polls will appear on the homepage

**Review by rob-gordon — APPROVED** [review]
Minor things that would be good to change but ready to merge after you do

File: app/helpers.ts (hunk 1 above)
Comment (AmmDuncan) [reply app/helpers.ts:3]: That's true. Thanks.

**Pull request merged**